docs/post_installation/post_installation.md

```diff
@@ -0,0 +1,166 @@
+
+> Hornet version 0.5.x targets IOTA 1.0 mainnet network by default. Hornet version 0.6.x targets IOTA 1.5 (Chrysalis) mainnet network by default
+
+### Identifying a configuration file based on particular use cases
+There is a simple way how to recognize which configuration file targets which IOTA network:
+```bash
+cat config.json | jq "[.httpAPI?, .restAPI?]"
+```
+* `jq` command line json parser can be installed using `sudo apt install jq`
+
+IOTA 1.5 (Chrysalis network) provides the following rest API endpoints:
+```json
+[
+  null,
+  {
+    "jwtAuth": {
+      "enabled": false,
+      "salt": "HORNET"
+    },
+    "excludeHealthCheckFromAuth": false,
+    "permittedRoutes": [
+      "/health",
+      "/mqtt",
+      "/api/v1/info",
+      "/api/v1/tips",
+      "/api/v1/messages/:messageID",
+      "/api/v1/messages/:messageID/metadata",
+      "/api/v1/messages/:messageID/raw",
+      "/api/v1/messages/:messageID/children",
+      "/api/v1/messages",
+      "/api/v1/transactions/:transactionID/included-message",
+      "/api/v1/milestones/:milestoneIndex",
+      "/api/v1/milestones/:milestoneIndex/utxo-changes",
+      "/api/v1/outputs/:outputID",
+      "/api/v1/addresses/:address",
+      "/api/v1/addresses/:address/outputs",
+      "/api/v1/addresses/ed25519/:address",
+      "/api/v1/addresses/ed25519/:address/outputs",
+      "/api/v1/treasury"
+    ],
+    "whitelistedAddresses": [
+      "127.0.0.1",
+      "::1"
+    ],
+    "bindAddress": "0.0.0.0:14265",
+    "powEnabled": true,
+    "powWorkerCount": 1,
+    "limits": {
+      "bodyLength": "1M",
+      "maxResults": 1000
+    }
+  }
+]
+```
+
+IOTA 1.0 provides legacy rest API endpoints:
+```json
+[
+  {
+    "basicAuth": {
+      "enabled": false,
+      "username": "",
+      "passwordHash": "",
+      "passwordSalt": ""
+    },
+    "excludeHealthCheckFromAuth": false,
+    "permitRemoteAccess": [
+      "getNodeInfo",
+      "getBalances",
+      "checkConsistency",
+      "getTipInfo",
+      "getTransactionsToApprove",
+      "getInclusionStates",
+      "getNodeAPIConfiguration",
+      "wereAddressesSpentFrom",
+      "broadcastTransactions",
+      "findTransactions",
+      "storeTransactions",
+      "getTrytes"
+    ],
+    "permittedRoutes": [
+      "healthz"
+    ],
+    "whitelistedAddresses": [],
+    "bindAddress": "0.0.0.0:14265",
+    "limits": {
+      "bodyLengthBytes": 1000000,
+      "findTransactions": 1000,
+      "getTrytes": 1000,
+      "requestsList": 1000
+    }
+  },
+  null
+]
+```
+
+## Dashboard
+There is an admin dashboard available in Hornet (on port 8081) and it is enabled by default. It provides some useful information regarding the node, its internal activity and its resources, such as memory, peers, transactions per second rate, etc.
+
+However it is not listening to incoming requests from a public traffic to prevent access from a malicious actor. It is listening only to requests from `localhost` by default.
```

Comment:
"from a public traffic" is not proper English.

Reply:
Changed